site/content/includes/installation/install-gateway-api-experimental-features.md

```diff
@@ -0,0 +1,23 @@
+---
+docs: "DOCS-000"
+---
+
+To use Gateway API experimental resources, the Gateway API resources from the experimental channel must be installed before deploying NGINX Gateway Fabric. Additionally, NGINX Gateway Fabric must have experimental features enabled.
+
+{{<caution>}}As noted in the [Gateway API documentation](https://gateway-api.sigs.k8s.io/guides/#install-experimental-channel), future releases of the Gateway API can include breaking changes to experimental resources and fields.{{</caution>}}
```

Comment:
@ADubhlaoich I see that its a bit inconsistent throughout our current documentation https://github.com/nginxinc/nginx-gateway-fabric/blob/b8c64df41a709ea84e4ecce46ffb9b91ee0a007a/site/content/includes/installation/install-gateway-api-resources.md?plain=1#L5 as an example, do you think this would be something worth standardizing and fixing in our documentation?

Reply:
It's not critical so I wouldn't dedicate time to it alone: for those kinds of small changes, I favour making them if I happen to be updating a given page. It displays the same to the user, so it's purely an internal style concern.